website/docs/r/db_instance.html.markdown

```diff
@@ -86,7 +86,9 @@ for more information.
 Default is `false`.
 * `db_subnet_group_name` - (Optional) Name of DB subnet group. DB instance will
 be created in the VPC associated with the DB subnet group. If unspecified, will
-be created in the `default` VPC, or in EC2 Classic, if available.
+be created in the `default` VPC, or in EC2 Classic, if available. It needs to be
```

Comment:
Can you please qualify this "only if" sentence with `For creating read replica instances, it needs to`... I think otherwise this can be confusing when you're not working with read replicas.

Reply:
Thanks for spotting this. Fixed